Source/Urho3D/Graphics/OpenGL/OGLGraphics.cpp

```diff
@@ -500,6 +500,9 @@ bool Graphics::SetMode(int width, int height, bool fullscreen, bool borderless,
     CheckFeatureSupport();
 
 #ifdef URHO3D_LOGGING
+    String adapterDesc = String((const char *) (glGetString(GL_VENDOR))) + " " + (const char *) (glGetString(GL_RENDERER));
+    URHO3D_LOGINFO("Adapter used " + adapterDesc);

```

Comment:
May be better to use `URHO3D_LOGINFOF()` with "Adapter used %s %s" as the formatter and the two `glGetString()` as paramaters.